sources/packages/backend/apps/db-migrations/src/sql/Types/Add-relationship-status-married-unable.sql

```diff
@@ -0,0 +1,19 @@
+-- Recreate the enums types when the new item must be added.
+CREATE TYPE sims.relationship_status_to_be_updated AS ENUM (
+  'married',
+  'single',
+  'other',
+  'marriedUnable'
```

Comment:
The AC specifies the value as `married unable` with a space. 
![image](https://github.com/user-attachments/assets/44752db3-f8ba-4c6f-9d19-db30a09f5c30)

Reply:
In general, we do not use dropdown values with white spaces, doing this for these seems against what we did for others.
Business can have an opinion about how it can potentially be displayed to the user, but how we persist will be a technical decision.
My understanding on the "provided AC value" was a suggestion on how to shortter the pretty long option.

Reply:
I asked the same as this information goes raw to SDPR response as well.

Reply:
It does not seem to be in the scope or planned for this ticket, but it would be worth asking.

Reply:
Business updated the AC to state: `marriedUnable`.